# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: dom/ipc/WindowGlobalParent.cpp
# Commit: 3b11a30f4146
# Full Hash: 3b11a30f41465b2f5de85bc5ff403d30795f6ca3
# Author: Tom Ritter <tom@mozilla.com>
# Date: 2024-03-23 09:29:17
# Description:
#   Bug 1886922: Do not dereference a Maybe if it is not Some r=mccr8
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D205409
# ==============================================================================

diff -r 16211a33b193 -r 3b11a30f4146 dom/ipc/WindowGlobalParent.cpp
--- a/dom/ipc/WindowGlobalParent.cpp	Fri Mar 22 15:13:58 2024 +0000
+++ b/dom/ipc/WindowGlobalParent.cpp	Fri Mar 22 15:32:39 2024 +0000
@@ -566,7 +566,7 @@
     stack->BorrowFromClonedMessageData(*aStack);
   }
   MMPrinter::Print("WindowGlobalParent::RecvRawMessage", aMeta.actorName(),
-                   aMeta.messageName(), *aData);
+                   aMeta.messageName(), aData);
   ReceiveRawMessage(aMeta, std::move(data), std::move(stack));
   return IPC_OK();
 }
